web/src/features/rbac/server/membersRouter.ts

```diff
@@ -0,0 +1,818 @@
+import { auditLog } from "@/src/features/audit-logs/auditLog";
+import {
+  createTRPCRouter,
+  protectedOrganizationProcedure,
+  protectedProjectProcedure,
+} from "@/src/server/api/trpc";
+import { TRPCError } from "@trpc/server";
+import * as z from "zod";
+import {
+  hasOrganizationAccess,
+  throwIfNoOrganizationAccess,
+} from "@/src/features/rbac/utils/checkOrganizationAccess";
+import { paginationZod, type PrismaClient, Role } from "@langfuse/shared";
+import { sendMembershipInvitationEmail } from "@langfuse/shared/src/server";
+import { env } from "@/src/env.mjs";
+import { hasEntitlement } from "@/src/features/entitlements/server/hasEntitlement";
+import {
+  hasProjectAccess,
+  throwIfNoProjectAccess,
+} from "@/src/features/rbac/utils/checkProjectAccess";
+
+// Record as it allows to type check that all roles are included
+const orderedRoles: Record<Role, number> = {
```

Comment:
I would define this in the file where we define roles as well.

Reply:
only used here, I'd keep it here as important as context for the next two functions